hardware_drivers/navigator_kill_board/CMakeLists.txt

```diff
@@ -9,4 +9,8 @@ catkin_python_setup()
 
 catkin_package(
 )
+if(CATKIN_ENABLE_TESTING)
+  find_package(rostest REQUIRED)
+  add_rostest(kill_board.test)
```

Comment:
needs to be ```test/kill_board.test```. You NEED to be able to successfully run
```catkin_make run_tests_navigator_kill_board```, as this is what the CI does.